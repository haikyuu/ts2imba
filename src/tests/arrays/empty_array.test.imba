import {build} from "../../index"
import * as imbac from 'imba/compiler'

let tsx-code = '''
var a = []
'''
test 'empty_array' do
	const result = await build tsx-code
	expect(result.code).toMatchSnapshot()
	const out = imbac.compile(result.code, sourceId: 'sth')
	expect(out.js).toBeDefined()
