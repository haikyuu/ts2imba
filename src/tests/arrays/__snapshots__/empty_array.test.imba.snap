// Vitest Snapshot v1

exports[`empty_array 1`] = `
"let a = []
"
`;
